src/app/pages/LiquidityMining/components/AddLiquidityDialog/index.tsx

```diff
@@ -133,6 +134,7 @@ export const AddLiquidityDialog: React.FC<IAddLiquidityDialogProps> = ({
                 tx.loading || !valid || !canInteract || addliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-deposit-button"
```

Comment:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```
Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals